blakserv/synched.c

```diff
@@ -662,6 +662,17 @@ void SynchedGotoGame(session_node *s,int last_download_time)
 
    // All set to go to game mode.
 
+   move_interval = ConfigInt(UPDATE_MOVE_INTERVAL);
+   // Sanity check to ensure move_interval isn't 0 or negative
+   if (move_interval <= 0)
+   {
+      move_interval = DEFAULT_MOVE_INTERVAL; // Use a default value if invalid
+   }
+
+   AddByteToPacket(AP_MOVE_INTERVAL);

```

Comment:
Are you sure you need to introduce a new message type for this?  Could you just add the move interval value to the existing AP_GAME message instead (the very next lines)?